tests/protos/controllers/modify_proto_template_field/.gitignore

```
@@ -0,0 +1 @@
+modify_proto_template_field
```

```suggestion
/modify_proto_template_field
```

Just to be coherent with the other gitignore.